install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -103,7 +103,7 @@ spec:
         command:
         - cilium-agent
         args:
-        - --config-dir=/tmp/cilium/config-map
+        - --config-dir=/tmp/cilium/config-final
```

Comment:
nit: this change is needed because `/tmp/cilium/config-final` is the `build-config.go` default dest right? If correct, how about either a comment or explicitly `--dest` a few lines below?